--- ./meson.build	2025-02-22 16:11:38.721689700 +0100
+++ ./meson.build	2025-02-22 16:12:14.123690100 +0100
@@ -16,16 +16,8 @@
   cpp_args: fmt_private_cpp_args,
   include_directories: 'include',
   build_by_default: not header_only,
-  install: not header_only,
 )
 
-if not header_only
-  install_subdir('include',
-    strip_directory: true,
-    install_dir: get_option('includedir'),
-  )
-endif
-
 fmt_dep = declare_dependency(
   include_directories: 'include',
   compile_args: fmt_interface_cpp_args,
@@ -40,11 +32,3 @@
   include_directories: 'include',
   compile_args: '-DFMT_HEADER_ONLY',
 )
-
-if not header_only
-  pkg_mod = import('pkgconfig')
-  pkg_mod.generate(fmt_lib,
-    name: 'fmt',
-    filebase: 'fmt',
-  )
-endif
