--- ./meson.build	2024-05-30 21:16:54.000000000 +0200
+++ ./meson.build	2025-02-22 15:47:43.372673200 +0100
@@ -98,7 +98,6 @@
   version: png_libversion,
   dependencies: libpng_deps,
   c_args: c_args,
-  install: true,
 )
 
 pngincsubdir = 'lib' + png_libname
@@ -110,13 +109,6 @@
   copy: true,
 )
 
-install_headers(
-  'png.h',
-  'pngconf.h',
-  pkglibconf,
-  subdir: pngincsubdir,
-)
-
 cdata = configuration_data()
 cdata.set('prefix', get_option('prefix'))
 cdata.set('exec_prefix', get_option('prefix'))
@@ -132,8 +124,6 @@
   input: 'libpng.pc.in',
   output: 'libpng16.pc',
   configuration: cdata,
-  install_dir: get_option('libdir') / 'pkgconfig',
-  install: true,
 )
 
 # FIXME: this should be a symlink to libpng16.pc
@@ -141,8 +131,6 @@
   input: 'libpng.pc.in',
   output: 'libpng.pc',
   configuration: cdata,
-  install_dir: get_option('libdir') / 'pkgconfig',
-  install: true,
 )
 
 libpng_dep = declare_dependency(
@@ -153,6 +141,3 @@
 # Keep the older dependency name for backwards-compat with old-style
 # dependency(..., fallback ['libpng': 'png_dep'])
 png_dep = libpng_dep
-
-png_test = executable('pngtest', 'pngtest.c', dependencies: libpng_dep)
-test('pngtest', png_test, args: files('pngtest.png'))
