--- ./meson.build	2025-02-21 16:32:45.570831800 +0100
+++ ./meson.build	2025-02-24 09:45:26.542289000 +0100
@@ -65,7 +65,7 @@
   input: 'modules.cfg',
   command: [python_exe, files('builds/meson/parse_modules_cfg.py'),
             '--format=ftmodule.h', '@INPUT@', '--output', '@OUTPUT@'],
-  install: true,
+  install: false,
   install_dir: join_paths(freetype_includedir, 'freetype/config'),
 )
 ft2_sources = [ftmodule_h]
@@ -367,7 +367,7 @@
   input: 'include/freetype/config/ftoption.h',
   output: 'ftoption.h',
   command: ftoption_command,
-  install: true,
+  install: false,
   install_dir: join_paths(freetype_includedir, 'freetype/config'),
 )
 ft2_sources += ftoption_h
@@ -395,7 +395,7 @@
     input: ftconfig_h_in,
     output: 'ftconfig.h',
     command: ftconfig_command,
-    install: true,
+    install: false,
     install_dir: join_paths(freetype_includedir, 'freetype/config'),
   )
   ft2_sources += ftconfig_h
@@ -409,7 +409,7 @@
   gnu_symbol_visibility: 'hidden',
   include_directories: ft2_includes,
   dependencies: ft2_deps,
-  install: true,
+  install: false,
   version: ft2_so_version,
   link_args: common_ldflags,
 )
@@ -424,33 +424,6 @@
 
 meson.override_dependency('freetype2', freetype_dep)
 
-
-# NOTE: Using both `install_dir` and `subdir` doesn't seem to work
-# below, i.e., the `subdir` value seems to be ignored, contrary to
-# examples in the Meson documentation.
-install_headers('include/ft2build.h',
-  install_dir: freetype_includedir)
-install_headers(ft2_public_headers,
-  install_dir: join_paths(freetype_includedir, 'freetype'))
-install_headers(ft2_config_headers,
-  install_dir: join_paths(freetype_includedir, 'freetype/config'))
-
-
-install_data('builds/unix/freetype2.m4',
-  install_dir: freetype_aclocaldir)
-
-
-pkgconfig = import('pkgconfig')
-
-pkgconfig.generate(ft2_lib,
-  filebase: 'freetype2',
-  name: 'FreeType 2',
-  description: 'A free, high-quality, and portable font engine.',
-  url: 'https://freetype.org',
-  subdirs: 'freetype2',
-  version: ft2_pkgconfig_version,
-)
-
 if get_option('tests').enabled()
   subdir('tests')
 endif
