--- ./src/lib/libdwarf/meson.build	2024-06-05 01:32:54.000000000 +0200
+++ ./src/lib/libdwarf/meson.build	2025-02-24 09:56:51.714219200 +0100
@@ -131,7 +131,7 @@
   dependencies : [ zlib_deps, libzstd_deps ],
   gnu_symbol_visibility: 'hidden',
   include_directories : config_dir,
-  install : true,
+  install : false,
   pic : true,
   version : meson.project_version()
 )
@@ -143,7 +143,3 @@
 )
 
 meson.override_dependency('libdwarf', libdwarf)
-
-install_headers(libdwarf_header_src,
-  install_dir : dir_pkginclude + '-' + v_maj
-)
